launch_ros/launch_ros/actions/node.py

```diff
@@ -72,12 +70,9 @@ class Node(ExecuteProcess):
     def __init__(
         self, *,
         executable: Optional[SomeSubstitutionsType] = None,
```

Comment:
executable should not be optional.

Reply:
nevermind, addressed in https://github.com/ros2/launch_ros/pull/197/
Sorry for the noise.